app/src/main/java/ar/com/wolox/android/example/di/module/MiscModule.kt

```diff
@@ -0,0 +1,14 @@
+package ar.com.wolox.android.example.di.module
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import dagger.Module
+import dagger.Provides
+
+@Module
+class MiscModule {
+
+    @Provides
+    internal fun provideBasePresenter(): BasePresenter<Any> {
```

Comment:
Is this done through a `Provides` for any particular reason? I believe you can `@Inject constructor()` unless i'm missing something.

Reply:
Just to be an example of how to use it. (It was before in the Java code)